service/history/workflow/update/registry.go

```diff
@@ -234,32 +226,35 @@ func (r *registry) TryResurrect(ctx context.Context, acptOrRejMsg *protocolpb.Me
 
 	// Check only total limit here. This might add more than maxInFlight updates to registry,
 	// but provides better developer experience.
-	if err := r.checkTotalLimit(ctx); err != nil {
+	if err := r.checkTotalLimit(); err != nil {
 		return nil, err
 	}
 
 	body, err := acptOrRejMsg.Body.UnmarshalNew()
 	if err != nil {
 		return nil, invalidArgf("unable to unmarshal request: %v", err)
 	}
+
 	err = utf8validator.Validate(body, utf8validator.SourceRPCRequest)
 	if err != nil {
 		return nil, invalidArgf("unable to validate utf-8 request: %v", err)
 	}
+
 	var reqMsg *updatepb.Request
 	switch updMsg := body.(type) {
 	case *updatepb.Acceptance:
 		reqMsg = updMsg.GetAcceptedRequest()
 	case *updatepb.Rejection:
 		reqMsg = updMsg.GetRejectedRequest()
+	default:
 		// Ignore all other message types.
 	}
 	if reqMsg == nil {
 		return nil, nil
 	}
 	reqAny, err := anypb.New(reqMsg)
 	if err != nil {
-		return nil, invalidArgf("unable to unmarshal request body: %v", err)
```

Comment:
This is marshalling, not unmarhalling.

Reply:
I think I renamed it in few places already.